packages/protocol/socket.ts

```diff
@@ -12,6 +12,7 @@ import {
   analysisPoints,
   analysisResult,
 } from "@replayio/protocol";
+import * as Sentry from "@sentry/react";
```

Comment:
:hand: We probably _shouldn't_ import the _entire_ Sentry library - just `captureException` by itself.